### ISSUE TITLE
Handle error 

### ISSUE THREAD

**Issue by geofranzi** [bexis/Module_EventManagement#36]
e.g. wrong email format

**Comment by EleonoraPetzold** [comment]
das wird noch dann mit dem javascript abgefangen, oder? siehe #31

**Comment by geofranzi** [comment]
Ja, müsste aber wird nicht. Allerdings da kommt nur einen 500 zurück aber damit wird nichts gemacht. Eine 500 kommt auch wenn die Buchung nicht geändert / neu angelegt werden kann, weil die email Adresse gleich ist usw..Auch wenn die Mail nicht verschickt werden konnte weil die email Adresse nicht valid war .. gibt evt sogar noch mehr Fälle. Statt 500 müsste ein false zurück kommen und im js eine passende Meldung angezeigt werden.

**Comment by EleonoraPetzold** [comment]
es ist ja noch kein JS an dem event, das müssen wir uns noch mal anschauen. ich weiß auch nicht wie weit du warst das zu implementieren und zu testen? Es wäre gut zu unseren event templates die im svn (_BERepository\ProjectFiles\Event Tool Schema Templates) liegen auch JSs zu haben die diese validieren. beides muss dann wie auch in B1 für jeden event angepasst werden.

**Comment by geofranzi** [comment]
Die Fehler abfangen gehört in die Form.js und in den Code. Das ist was anderes also die zusätzliche js für die Zusätzliche inhaltliche Validierung.

**Comment by EleonoraPetzold** [comment]
validierung ist soweit drin in der neuen version.
